### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by WinnyTroy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,35 @@
-Role Name
+Ansible-kpi
```
Comment (jasonrogena) [inline README.md]: @WinnyTroy please add the TravisCI build badge here.

File: README.md [hunk 2]
```diff
@@ -1,23 +1,35 @@
...
-Requirements
+Role Dependencies
 ------------
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the boto package is required.
+- onaio.collectd
+- onaio.monit
+- onaio.postgresql
```
Comment (jasonrogena) [inline README.md]: Not a direct requirement. We only use it in the tests.

File: README.md [hunk 3]
```diff
@@ -1,23 +1,35 @@
...
+- ANXS.python
+- onaio.nvm
+- onaio.django
+- Stouts.backup
```
Comment (jasonrogena) [inline README.md]: Not a requirement.

File: README.md (hunk 2 above)
Comment (WinnyTroy) [reply README.md]: oh...let me remove that

**Pull request merged**